app/frontend/packs/sortable-table.js

```diff
@@ -0,0 +1,239 @@
+const SortableTable = (table) => {
```

Comment:
I've copied and pasted all this JS but it still needs some tidying up (es6 syntax, linting)